src/api/actions/initialize.ts

```diff
@@ -57,4 +59,11 @@ const verify = async (authToken: string): Promise<auth_status> => {
   return auth;
 };
 
+export const firebaseApp = initializeApp({
+  apiKey: process.env.REACT_APP_GCP_APIKEY,
+  storageBucket: process.env.REACT_APP_GCP_BUCKET_URL,
+});
+
+export const firebaseStorage = getStorage(firebaseApp);
```

Comment:
Consider moving to a separate file for just firebase config. Perhaps somewhere like `src/firebase/config.ts` & other associated firebase code can all reside in the same place.

Reply:
Probably will do this. I figured since it's a temporary fix (this will be in the backend once we migrate) it'd be fine, but I'll move it anyways just to be safe.